cg/constants/nextflow.py

```diff
@@ -12,6 +13,7 @@
 NXF_PID_FILE_ENV = "NXF_PID_FILE"
 NXF_JVM_ARGS_ENV = "NXF_JVM_ARGS"
 JAVA_MEMORY_HEADJOB = "-Xmx5g"
+NXF_WORKFLOWS = [Workflow.RAREDISEASE, Workflow.RNAFUSION, Workflow.TAXPROFILER, Workflow.TOMTE]
```

Comment:
I don't like the NXF abreviations here and in the constants above, doesn't help with readability in my opinion

```suggestion
NEXTFLOW_WORKFLOWS = [Workflow.RAREDISEASE, Workflow.RNAFUSION, Workflow.TAXPROFILER, Workflow.TOMTE]
```

Reply:
NXF_JVM_ARGS_ENV and co are very readable for people used to the nextflow environment as they are the exact same spelling, but NEXTFLOW_WORKFLOWS or NF_WORKFLOW is good for me

Reply:
What I mean is it makes the search maybe a tad easier if you search for the env variable name

Reply:
Thanks for the explanation, it makes sense to have them defined in the same way as in nextflow 👍

Reply:
Regarding `nf_` vs `nextflow_`, I would say we should stick with `nextflow_`, since I've also seen this refactored recently from `nf_`

Reply:
Yes, I have changed to NEXTFLOW